updating/index.adoc

```diff
@@ -6,52 +6,52 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4.
+You can update an {product-title} 4 cluster with a single operation by using the web console or the OpenShift CLI (oc).
 
 [id="updating-clusters-overview-understand-update-service"]
 == Understanding the OpenShift Update Service
-xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red{nbsp}Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, see the following information:
+xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, see the following:
 
 * xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[About the OpenShift Update Service]
 * xref:../updating/understanding-the-update-service.adoc#unmanaged-operators_understanding-the-update-service[Understanding support policy for unmanaged Operators]
 
 [id="updating-clusters-overview-install-and-configure-update-service"]
 == Installing and configuring the OpenShift Update Service
 xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]: Clusters with internet accessibility can access public APIs; clusters in a restricted network are unable to access public APIs for update information. To provide a similar upgrade experience in a restricted network, you can install and configure the OpenShift Update Service locally so that it is available within a disconnected environment.
-For more information, see the following actions:
+For more information, see the following procedures:
```

Comment:
Personal preference: I'd write "...see the following:"

Reply:
Addressed